elections/steering/2022/README.md

```diff
@@ -79,6 +79,9 @@ Eligibility for voting in 2021 is defined as:
   Contributions include GitHub events like creating issues, creating PRs,
   reviewing PRs, commenting on issues, etc. For full details see
   [the SQL query used by devstats for developer activity counts][devstats-sql].
+  
+* Members of the Code of Conduct Committee (CoCC) and Security Response Committee
```

Comment:
"Members of..." is potentially ambiguous here.  I think it should be more explicit like "Individuals listed as leadership chairs in sigs.yaml..."

Reply:
SRC has associates too, it would be good make clear if they are included

Reply:
I think you're commenting on the wrong PR.  There are no leadership chairs here.

This one is just CoCC and SRC.  The listing for those two committees in SIGs.yaml is not in any way ambiguous.  CoCC has 5 members and SRC has 7, and nobody else is listed.  I can see adding "as listed in sigs.yaml" as a good confirmer.

Reply:
> I think you're commenting on the wrong PR. There are no leadership chairs here.

The SRC has [7 full committee members and 5 associate members](https://github.com/kubernetes/committee-security-response#security-response-committee-src), I was just asking for clarification if they are included.

Reply:
> I can see adding "as listed in sigs.yaml" as a good confirmer.

With that made explicit, this PR looks good to me

Reply:
Micah: what do the associate members do?  I'm wondering if they should be included, or not.

Reply:
Associate responsibilities are described in https://github.com/kubernetes/committee-security-response/blob/main/security-release-process.md#associate, and are oriented towards public / non-disclosure work, which (in my opinion) is more visible and more likely to be covered via the normal devstats mechanism.

Reply:
Is it possible to be an Associate and do very little/nothing all year?

Reply:
I'll defer to Micah, but I suspect it's possible... if that's the case, maybe low devstats activity is an accurate reflection of little/no involvement, and collapsing to the normal threshold for contributor activity / voting eligibility seems ok to me

Reply:
or, at least, it suggests that Associate Members should still need to file exceptions.

Reply:
> or, at least, it suggests that Associate Members should still need to file exceptions.

right